app/component-data/NavigationBar/joinFlowTabData.tsx

```diff
@@ -2,6 +2,7 @@ import EcosystemIcon from "~/ui/design-system/images/content/ecosystem"
 import AmbassadorsIcon from "~/ui/design-system/images/nav/ambassadors"
 import SocialsIcon from "~/ui/design-system/images/nav/socials"
 import { Section } from "~/ui/design-system/src/lib/Components/NavigationBar/types"
+import { DISCORD_URL } from "~/ui/design-system/src/lib/constants"
```

Comment:
Constants that are part of the site should not be defined in the design system.
Move this to `/app/constants/index.ts`

Reply:
Ty!